docling_core/types/doc/document.py

```diff
@@ -1178,6 +1116,98 @@ def export_to_doctags(
         return text
 
 
+class TableCell(NodeItem):
+    """TableCell."""
+
+    bbox: Optional[BoundingBox] = None
+    row_span: int = 1
+    col_span: int = 1
```

Comment:
Do we need a variable here? I think this can be a method computing `end_col_offset_idx-start_col_offset_idx`?

Reply:
We can do something about this but it is unrelated to the changes of this PR, and we cannot break backwards-compatibility.